### PR TITLE
Setup and mobile first

### PR DESCRIPTION
**Pull request by danielf-canon**
In this current branch these are some of the features recently added:

- The section navbar of my portfolio
- Main introductoy section.

### CODE REVIEW

**Review by NatiG25 — COMMENTED** [review]
## :red_circle: INVALID PULL REQUEST :red_circle: 

## Hi @danielf-canon, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that <EXPLAIN WHY PROJECT IS INVALID>
:x: Didn't have proper README:
- Didn't remove Microverse Template.
- Not having an MIT license. 
- Not including all main headers from the [REAMDE template](https://github.com/microverseinc/readme-template) 
- Having a Live Demo section without having a deployed project. Since it's not mandatory to have your project deployed, kindly, remove it.
- Not removing the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed.

Screenshot from 2023-05-04 23-56-18

:x: Not having a proper PR description.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Johnadibe — COMMENTED** [review]
Hi @danielf-canon, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that has a broken MIT License link in your README file. To address this issue, I would recommend creating a LICENSE.md file in your repository and also include the license header at the top of the file. Then, you can copy the text of the MIT License into this file.👍

license

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+</head>
+<body>
+  <header>
+    <nav>
+      <div>
+        <ul class="shortcut-bar">
+          <li><a href="#projects">Portfolio</a></li>
+          <li><a href="#About-me">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
```
Comment (stanleySimeon) [inline index.html:19]: Kindly I suggest you remove the menu list in the mobile version of your design, it should appear only in the desktop version

>### Your design
>
>
Screenshot 2023-05-05 at 11 12 58 PM

File: index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  <main>
+    <section class="container">
+      <div>
+        <h1>Hey there. I'm Daniel.</h1>
+        <p id="presentation">I'm a software developer</p>
+      </div>
+      <div>    
+        <p>I can help you build a product , feature or website Look through  some of my work and experience! <br> If you like what you see and have  a project you need coded, 
+        don't hesitate to contact me.</p>
+      </div>
```
Comment (stanleySimeon) [inline index.html:38]: Kindly you should stick to the design as much as possible using the same (font, margin, padding, colors, etc.), I suggest you use the same font in the Figma design.

>### Your design
>
>Screenshot 2023-05-05 at 11 18 35 PM
>
>### The Figma design
>
>Screenshot 2023-05-05 at 11 19 34 PM
>
>### Kindly make sure you follow the Figma template and make sure you use the correct layout before asking for another review

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @danielf-canon 
 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

# Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers, and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @stanleySimeon in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  padding-left: 1rem;
+  padding-right: 1rem;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+nav {
+  display: flex;
+  justify-content: end;
+}
```
Comment (emmiiorji) [inline style.css:28]: You have done great by removing the menu items on mobile version, kindly adjust the background image not to overlap with the hamburger image.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @danielf-canon, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<div align='center'>

</div>

 ## Good Highlights✔️

- Good use of background image to represent non-content image
- Good documentation in readme


 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @danielf-canon , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Nice UI✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @danielf-canon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**